pkg/util/helm/bundle.go

```diff
@@ -9,11 +9,17 @@ import (
 	"github.com/helm/helm/pkg/chartutil"
 )
 
+type HelmChart struct {
+	ChartFile []byte
+	Templates map[string]string
+	Values []byte
+}
+
 // ToBundle converts the helm chart on disk into a Kudo Bundle
-func ToBundle(folder string) (bundle.Framework, error) {
+func ToBundle(helmChart *HelmChart) (bundle.Framework, error) {
```

Comment:
this is the main change, now this method is better testable because it accepts parsed chart and produces our package also in memory...